examples/rust/robby_fischer/README.md

```diff
@@ -0,0 +1,317 @@
+<!--[metadata]
+title = "Robby fischer"
+tags = ["3D", "URDF", "Blueprint"]
+source = "https://github.com/02alexander/robby-fischer/tree/urdf-vis"
+thumbnail = "https://static.rerun.io/robby_thumbnail/71d2d57e9720e7a96e35a43467b5d2c45aa716d9/480w.png"
+thumbnail_dimensions = [480, 385]
+-->
+
+https://vimeo.com/989548054?autoplay=1&loop=1&autopause=0&background=1&muted=1&ratio=1920:1080
+
+## Used Rerun types
+
+[`Transform3D`](https://www.rerun.io/docs/reference/types/archetypes/transform3d), [`Points2D`](https://www.rerun.io/docs/reference/types/archetypes/points2d), [`Boxes3D`](https://www.rerun.io/docs/reference/types/archetypes/boxes3d), [`Pinhole`](https://www.rerun.io/docs/reference/types/archetypes/pinhole), [`Image`](https://www.rerun.io/docs/reference/types/archetypes/image), [`Mesh3D`](https://www.rerun.io/docs/reference/types/archetypes/mesh3d), [`LineStrips3D`](https://www.rerun.io/docs/reference/types/archetypes/line_strips3d),
+
+## Background
+Robby Fischer is an autonomous robot arm that you can play chess against, created by Alexander Berntsson and Herman Lauenstein. It detects the human's move by using a camera that watches which squares has a piece on it and what color that piece is. It doesn't need to see recognize different roles (pawn, rook, etc..) visually because it knows the start position so it can always figure out what piece stand on what square, although it's a bit problematic if the human promotes a pawn because robot must figure out which piece the pawn promoted to. This is why it also looks at the adjacent white board, it has a specific location associated with each piece, so if the human promotes to a queen the queen square will be empty and Robby can figure out that the human promoted to a queen.
+
+To find out if a piece stands on a square we must determine what part of the image that may only contain the piece that stands on that square. This is necessary to deal with the fact that some pieces are tall and blocks part of adjacent squares, e.g if a king stands on e2, it's head will block part of the e1 square in the image. The mask that determines it is logged to "images/mask" and is shown in the bottom left corner along with the detected pieces.
```

Comment:
```suggestion
To find out if a piece stands on a square we must determine what part of the image may only contain the piece that stands on that square. This is necessary to deal with the fact that some pieces are tall and block part of adjacent squares, e.g. if a king stands on `e2`, its head will block part of the `e1` square in the image. The mask that determines this is logged to `images/mask` and is shown in the bottom left corner along with the detected pieces.
```

Example picture after this might be nice :)